pkgs/development/python-modules/june-analytics-python/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  dateutils,
+  requests,
+  monotonic,
+  backoff,
+}:
+
+buildPythonPackage {
+  pname = "june-analytics-python";
+  version = "unstable-2022-07-26";
```

Comment:
https://github.com/juneHQ/analytics-python/releases/tag/2.2.0 looks newer than this